src/types/i18n.ts

```diff
@@ -0,0 +1,146 @@
+// todo: check I18nConfig once we have settled on a more robust language dictionary structure
```

Comment:
Can we not leverage Type Augmentation and Merging Interfaces? https://www.i18next.com/overview/typescript

Reply:
We gebruiken een andere library ;). ```https://next-intl-docs.vercel.app/docs/workflows/typescript``` dit is een stuk gemakkelijker.